### ISSUE TITLE
PowerShell Preview in MS Store still on 7.2-rc.1

### ISSUE THREAD

**Issue by allan2** [PowerShell/PowerShell#16494]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

7.2.0 has been released but PowerShell Preview in the MS Store is still on 7.2-rc.1.

### Expected behavior

```console
Preview version should be greater or equal to the stable version.
```

### Actual behavior

```console
N/A
```

### Error details

```console
N/A
```

### Environment data

```powershell
N/A
```

**Comment by iSazonov** [comment]
I guess we should wait 1-2 week after release.

**Comment by daxian-dbw** [comment]
PowerShell Preview and PowerShell are 2 different channels (apps) in MS Store. The former only publish preview releases, and the latter only for stable releases,

**Comment by allan2** [comment]
It just seemed odd to me that preview is behind stable, but I guess that makes sense if there's no new preview.

**Comment by daxian-dbw** [comment]
Yeah, a new preview will be out in December (I could be wrong on the schedule), and that will be published to the "PowerShell Preview" channel only.

**Comment by NatoBoram** [comment]
So you guys are telling me that 

>    A new PowerShell stable release is available: v7.2.0
>   Upgrade now, or check out the release page at:
>     https://aka.ms/PowerShell-Release?tag=v7.2.0

is the normal and expected experience from PowerShell Preview?

I've never seen a software where its "preview" channel is so disconnected from the main product that it doesn't receive stable updates. It's just fundamentally wrong -- unless you actually want to decommission the channel entirely.

**Comment by iSazonov** [comment]
@NatoBoram Please open new issue with your proposal.
